src/main/java/com/fasterxml/jackson/databind/util/ReflectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.fasterxml.jackson.databind.util;
+
+import sun.reflect.ReflectionFactory;
```

Comment:
This is not an official part of JDK is it?

I don't think this code can be added directly in `jackson-databind` but needs to go in a module -- it could become one of "Base" modules, if this class is available on wide enough range of JDKs/JREs.